lib/async/http/faraday/adapter.rb

```diff
@@ -148,6 +148,20 @@ def with_timeout(task: Async::Task.current)
 						yield
 					end
 				end
+				
+				def encoded_body(res)
```

Comment:
Please avoid abbreviations:

```suggestion
				def encoded_body(response)
```